packages/esm-patient-medications-app/src/order-basket/order-basket.component.tsx

```diff
@@ -95,12 +101,12 @@ const OrderBasket = connect<OrderBasketProps, OrderBasketStoreActions, OrderBask
       orderEncounterUuid = await createEmptyEncounter(
         patientUuid,
         config?.drugOrderEncounterType,
-        currentVisit?.uuid,
+        activeVisitRequired ? currentVisit?.uuid : null,
```

Comment:
What happens if we have an existing visit but `activeVisitRequired` is `false`?

Reply:
`activeVisitRequired` is the global setting in the legacy UI which the admin has already disabled, and hence we don't consider visits at all.